integration_tests/service_a/src/schema.py

```diff
@@ -12,11 +12,17 @@ class FunnyText(ObjectType):
     id = external(Int(required=True))
 
 
+@extend(fields='name')
+class MultipleKeys(ObjectType):
```

Comment:
```suggestion
class User(ObjectType):
```
Better name it User (or another domain-related name) and add 2 primary keys: `id` & `email`
to make it more intuitive for understanding